ntbs-service/Pages/Notifications/Overview.cshtml.cs

```diff
@@ -29,7 +29,7 @@ public async Task<IActionResult> OnGetAsync(int id)
             await AuthorizeAndSetBannerAsync();
             if (!HasEditPermission)
             {
-                return Partial("./UnauthorizedWarning", this);
+                return ForbiddenResult();

```

Comment:
Am I right in saying the partial used to show the banner? I'm guessing the forbidden result does not do that? If not, then I don't think we want that change - anyone on the system wants to be able to see the banner for each notification.
Where a proper `forbidden` page does want to be shown, is if the user is not part of the system at all (ie. no in the base group ("BaseUserGroup": "Global.NIS.NTBS", see appsettings.json).
And perhaps on edit pages such as `Notification/:id/Edit/Patient`, as they don't have edit rights.

Reply:
Whoops yes you're right, will put it back to how it was and just show the forbidden result for users with the wrong base group

Reply:
@jasiekmiko Can users without the correct Base Group even gain access to the site? Is there no restriction on the AD integration level?

Reply:
When a user isn't authorized with the correct role, they are redirected to Accounts/AccessDenied, which was already present, so I've just changed the styling of this page and removed 403.cshtml as we don't throw a Forbidden for unauthorized users anymore

Reply:
ah sorry for only picking up the question now, but seems like you've got the right answer here! :)
Is the `AccessDenied` also returned alongside 403 code? if not, are we able to do that?

Reply:
The default behaviour is for accessdenied to return a redirect (302) - I just tried getting it to return a 403, but couldn't quite get it to work - I will merge for now, but if you feel it's important for this to return a 403 I could raise a ticket?

Reply:
No, I think that's fine - it would have been a nice-to-have, but we aren't building an api, but a webpage where the content matters more than the codes :)